lib/aws/rails/sqs_active_job/executor.rb

```diff
@@ -36,8 +42,31 @@ def execute(message)
               job_msg = job ? "#{job.id}[#{job.class_name}]" : 'unknown job'
               @logger.info "Error processing job #{job_msg}: #{e}"
               @logger.debug e.backtrace.join("\n")
+            ensure
+              refresh_monitor = false
             end
           end
+
+          @monitor.post(message) do |message|
```

Comment:
will this blow up if `@visibility_refresh` is not set since `@monitor` will be `nil`